File: team1/Sort/byeongguk/Solution01.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

def solution(array, commands):
    answer = []

    for command in commands:
        temp = sorted(array[command[0] - 1:command[1]])
        answer.append(temp[command[2] - 1])

    return answer

File: study-2nd/level2/seungjin/solution01.java
class Solution {
    public int[] solution(int brown, int red) {
        int[] answer = new int[2];
        
        int x=0;
        int y=0;
        
        //x, y
        int add = (brown+4)/2;
        int mul = brown + red;
        
        for(int i=0; i<=add; i++){
            x = i;
            y =  (add-i);
            
            if(x*y == mul){
                break;
            }
        }
        
        if(x<y){
            int temp = x;
            x = y;
            y = temp;
        }
        
        answer[0] = x;
        answer[1] = y;
        
        return answer;
    }
}
